### PR TITLE
Added Front End

### PR DESCRIPTION
**Pull request by SonaliBedge**
Added Front End

### CODE REVIEW

File: public/addEdit.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+              schoolName.value = "";
+              grade.value = "";
+              subject.value = "All";
+
+              showStudents();
+            } else {
+              message.textContent = data.msg;
+            }
+          }
+        } catch (err) {
```
Comment (ethanarns) [inline public/addEdit.js:63]: Good error catching

**Review by ethanarns — APPROVED** [review]
The system works and I see no errors within the code. Well done!

File: public/addEdit.js (hunk 1 above)
Comment (SonaliBedge) [reply public/addEdit.js:63]: Many thanks

Comment (SonaliBedge) [reply public/addEdit.js:63]: Thank you!

**Pull request merged**